apps/session-manager/src/__mocks__/services/redisSessionStorageService.mocks.ts

```diff
@@ -9,20 +9,36 @@ import { AssertionRef } from "../../generated/lollipop-api/AssertionRef";
 
 import { anAssertionRef } from "../lollipop.mocks";
 
+type GetLollipopAssertionRefForUser =
+  (typeof RedisSessionStorageService)["getLollipopAssertionRefForUser"];
+type DelLollipopDataForUser =
+  (typeof RedisSessionStorageService)["delLollipopDataForUser"];
+type DeleteUser = (typeof RedisSessionStorageService)["deleteUser"];
```

Comment:
here we can use the alternative format I suggested in another comment, like `typeof RedisSessionStorageService.deleteUser`